hugovallada/ojbank
# Add repository, service and exceptions interceptor

## Status
**READY**

## Description
This pr should implement 

## Todos



## Steps to Test or Reproduce
Access http://localhost:8080/api/v1/clients/Hugo and you should see an error response

```sh
mvn spring-boot:run
http://localhost:8080/api/v1/clients/Hugo
```
